src/helpers/client.js

```diff
@@ -5,13 +5,10 @@ import { history } from '../helpers/history';
 
 require('dotenv').config();
 
-const requiresAuth = (url) => {
-  return (
-    url !== 'rest-auth/login/' &&
-    url !== 'rest-auth/registration/' &&
-    url !== 'rest-auth/logout/'
-  );
-};
+const requiresAuth = (url) =>
```

Comment:
You can define a file with routes and in each object add if is private or not, check this for example https://github.com/rootstrap/herddle-web/blob/7c4b7f3842905f43c2c6b708c4994c772bb19f73/src/routes.js

Reply:
Or you can define a constant like this 
```
const REQUIRES_ROUTES = ['rest-auth/login/', 'rest-auth/registration/' ...]
```
and in this function use `REQUIRES_ROUTES.indexOf(url) > -1`